src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -9,28 +9,48 @@
 import java.util.Random;
 
 public class FigureSupplier {
+    private static final int MAX_RANDOM_NUMBER = 20;
+    private static final int MAX_RANDOM_FIGURES = 5;
+    private static final int DEFAULT_RADIUS_LENGTH = 10;
+    private String randomColor = new ColorSupplier().getRandomColor();
+    private int firstRandomNumber = new Random().nextInt(MAX_RANDOM_NUMBER) + 1;
```

Comment:
Why do you create three variables with random number? You could create a method getRandomNumber(), and then reuse it. Is it necessary to keep those numbers in the variables?

Reply:
You told me to change it:
https://github.com/mate-academy/jv-oop-advanced/pull/1758#discussion_r1878206497
but it didn't work correctly.
Now I don't know how you want. 
When I have only one its not working correctly: upper base: 6.0 lower base: 6.0 height: 6.0.
Figure: IsoscelesTrapezoid, area: 36.0 sq. units, upper base: 6.0 lower base: 6.0 height: 6.0 units, color: black